frontend/src/components/cluster-dashboard/cluster-graph-view/ClusterGraphView.tsx

```diff
@@ -0,0 +1,233 @@
+import React, { useState, useEffect } from "react";
+import ReactDOMServer from "react-dom/server";
+import { Grid } from "@material-ui/core";
+import Graph from "react-graph-vis";
+import "./../../../../node_modules/vis-network/dist/vis-network.css";
+
+import "./ClusterGraphView.scss";
+import styles from "./ClusterGraphView.module.scss";
+import { Cluster } from "../Cluster.model";
+
+type ClusterGraphViewProps = {
+  clusterData: Cluster;
+};
+
+const ClusterGraphView: React.FC<ClusterGraphViewProps> = (
+  props: ClusterGraphViewProps
+) => {
+  const clusterSvg =
+    '<svg xmlns="http://www.w3.org/2000/svg" width="50" height="50">' +
+    '<foreignObject x="8.5" y="17" width="100%" height="100%">' +
+    '<div xmlns="http://www.w3.org/1999/xhtml" ' +
+    "style=\"font-size:10px; font-family:'Segoe UI'; font-weight:bold; font-style:italic\">" +
+    '<span style="color:' +
+    styles.primaryColor +
+    '">' +
+    " Cluster</span>" +
+    "</div>" +
+    "</foreignObject>" +
+    "</svg>";
+  const clusterUrl =
+    "data:image/svg+xml;charset=utf-8," + encodeURIComponent(clusterSvg);
```

Comment:
Still think template literals are better for this sort of thing because of javascript weird coersion. I guess it's not really a huge deal because of your typeings...